WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetails/ViewModel/RegisterDomainDetailsViewModel+CountryDialCodes.swift

```diff
@@ -0,0 +1,277 @@
+        "ID": 62,
+        "IE": 353,
+        "IL": 972,
+        "IM": 44,
+        "IN": 91,
+        "IO": 246,
+        "IQ": 964,
+        "IR": 98,
+        "IS": 354,
+        "IT": 39,
+        "JE": 44,
+        "JM": 1876,
+        "JO": 962,
+        "JP": 81,
+        "KE": 254,
+        "KG": 996,
+        "KH": 855,
+        "KI": 686,
+        "KM": 269,
+        "KN": 1869,
+        "KP": 850,
+        "KR": 82,
+        "KV": 383,
+        "KW": 965,
+        "KY": 1345,
+        "KZ": 7,
+        "LA": 856,
+        "LB": 961,
+        "LC": 1758,
+        "LI": 423,
+        "LK": 94,
+        "LR": 231,
+        "LS": 266,
+        "LT": 370,
+        "LU": 352,
+        "LV": 371,
+        "LY": 218,
+        "MA": 212,
+        "MC": 377,
+        "MD": 373,
+        "ME": 382,
+        "MF": 590,
+        "MG": 261,
+        "MH": 692,
+        "MK": 389,
+        "ML": 223,
+        "MM": 95,
+        "MN": 976,
+        "MO": 853,
+        "MP": 1670,
+        "MQ": 596,
+        "MR": 222,
+        "MS": 1664,
+        "MT": 356,
+        "MU": 230,
+        "MV": 960,
+        "MW": 265,
+        "MX": 52,
+        "MY": 60,
+        "MZ": 258,
+        "NA": 264,
+        "NC": 687,
+        "NE": 227,
+        "NF": 672,
+        "NG": 234,
+        "NI": 505,
+        "NL": 31,
+        "NO": 47,
+        "NP": 977,
+        "NR": 674,
+        "NU": 683,
+        "NZ": 64,
+        "OM": 968,
+        "PA": 507,
+        "PE": 51,
+        "PF": 689,
+        "PG": 675,
+        "PH": 63,
+        "PK": 92,
+        "PL": 48,
+        "PM": 508,
+        "PN": 64,
+        "PS": 970,
+        "PT": 351,
+        "PW": 680,
+        "PY": 595,
+        "QA": 974,
+        "RO": 40,
+        "RS": 381,
+        "RU": 7,
+        "RW": 250,
+        "SA": 966,
+        "SB": 677,
+        "SC": 248,
+        "SD": 249,
+        "SE": 46,
+        "SG": 65,
+        "SH": 290,
+        "SI": 386,
+        "SJ": 47,
+        "SK": 421,
+        "SL": 232,
+        "SM": 378,
+        "SN": 221,
+        "SO": 252,
+        "SR": 597,
+        "SS": 211,
+        "ST": 239,
+        "SV": 503,
+        "SX": 1721,
+        "SY": 963,
+        "SZ": 268,
+        "TA": 290,
+        "TC": 1649,
+        "TD": 235,
+        "TF": 262,
+        "TG": 228,
+        "TH": 66,
+        "TJ": 992,
+        "TK": 690,
+        "TL": 670,
+        "TM": 993,
+        "TN": 216,
+        "TO": 676,
+        "TR": 90,
+        "TT": 1868,
+        "TV": 688,
+        "TW": 886,
+        "TZ": 255,
+        "UA": 380,
+        "UG": 256,
+        "UM": 1,
+        "US": 1,
+        "UY": 598,
+        "UZ": 998,
+        "VA": 39,
+        "VC": 1784,
+        "VE": 58,
+        "VG": 1284,
+        "VI": 1340,
+        "VN": 84,
+        "VU": 678,
+        "WF": 681,
+        "WS": 685,
+        "YE": 967,
+        "ZA": 27,
+        "ZM": 260,
+        "ZW": 263]
+
+    func countryCodePrefix(for countryCode: String) -> Int? {
```

Comment:
Country Code is better represented with String so we can return String here

Reply:
Good point!